lib/coinbase/transfer.rb

```diff
@@ -132,13 +132,13 @@ def transaction_link
     # Returns the Unsigned Payload of the Transfer.
```

Comment:
Can we update the `transaction_link` to use the `transaction.transaction_link`?

Reply:
updated in most recent commit!